app/containers/message/Components/Attachments/Quote.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import { dequal } from 'dequal';
+import { View } from 'react-native';
+
+import { Reply } from './components';
+import MessageContext from '../../Context';
+import { IMessageAttachments } from '../../interfaces';
+import { IAttachment } from '../../../../definitions';
+import { getMessageFromAttachment } from '../../utils';
+
+const isValidAttachment = (file?: IAttachment): boolean => {
```

Comment:
Both quotes and files are valid attachments.
`isQuoteAttachment` makes more sense.